docker/nginx/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM nginx:latest as gewisweb_nginx
+FROM nginx:stable-alpine as gewisweb_nginx
+
+RUN adduser -D -H -u 1000 -s /bin/bash www-data -G www-data
```

Comment:
Same comment about the usage of bash instead of sh as on the PR for gewisdb. We should probably replace the occurrences in the code.
![afbeelding](https://user-images.githubusercontent.com/32361020/183240506-ab6f30c8-7bce-45c8-92ec-44052e6baf92.png)

Reply:
:/